packages/terser/src/index.bzl

```diff
@@ -0,0 +1,126 @@
+"User configuration to run the terser binary under bazel"
+
+load("@build_bazel_rules_nodejs//:providers.bzl", "JSModuleInfo", "collect_js_modules")
+
+TERSER_ATTRS = {
+    "srcs": attr.label_list(
```

Comment:
I commented earlier on changing this to just `src`, which I think makes the `collect_js_modules` unnecessary which might be nice. Any downside to doing that?

Reply:
I think `src` will likely be a better shape since it ends up being a single output file one way or the other